library/core/src/ptr/mod.rs

```diff
@@ -964,7 +968,7 @@ pub unsafe fn write<T>(dst: *mut T, src: T) {
 /// ```
 #[inline]
 #[stable(feature = "ptr_unaligned", since = "1.17.0")]
-#[rustc_const_unstable(feature = "const_ptr_write", issue = "none")]
+#[rustc_const_unstable(feature = "const_ptr_write", issue = "86302")]
```

Comment:
This one is new, `write_unaligned` was not `const fn` before https://github.com/rust-lang/rust/pull/86003.

Seems fine to me, but the PR description should call out that this is more than a partial revert of https://github.com/rust-lang/rust/pull/86003.

Reply:
Oh wait, this is juts adding a tracking issue. Never mind then. :)